testsuite/tests/lib-format/print_array.ml

```diff
@@ -0,0 +1,33 @@
+(* TEST
+   include testing
+*)
+
+(*
+
+A test file for the Format module.
+
+*)
+
+open Testing;;
+open Format;;
+
+let say s = Printf.printf s;;
```

Comment:
Minor nit, just for less noisy printf formats later on:
```suggestion
let say s = Printf.printf (s ^^ "\n%!");;
```

Reply:
I just pasted the one in `print_seq` but OK.